virttest/qemu_monitor.py

```diff
@@ -849,18 +849,25 @@ def set_link(self, name, up):
             status = self.on_str
         return self.cmd("set_link %s %s" % (name, status))
 
-    def live_snapshot(self, device, snapshot_file, snapshot_format="qcow2"):
+    def live_snapshot(self, device, snapshot_file, **kwargs):
         """
         Take a live disk snapshot.
 
-        :param device: device id of base image
-        :param snapshot_file: image file name of snapshot
-        :param snapshot_format: image format of snapshot
+        :param device: the name of the device to generate the snapshot from.
+        :param snapshot_file: the target of the new image. A new file will be created.
+        :param kwargs: optional keyword arguments to pass to func.
+        :keyword args (optional):
+            format: the format of the snapshot image, default is 'qcow2'.
 
-        :return: The response to the command
+        :return: The response to the command.
         """
-        cmd = ("snapshot_blkdev %s %s %s" %
-               (device, snapshot_file, snapshot_format))
+        device = device
+        snapshot_file = snapshot_file
+        if 'format' in kwargs:
```

Comment:
cmd = "snapshot_blkdev  %s %s" % (device, snapshot_file)
if 'format' in kwargs:
     cmds += " %s" % kwargs['format']
return self.cmd(cmd)